QA_CUSTOM_NETWORK.md

```diff
@@ -0,0 +1,80 @@
+# Dev Settings
```

Comment:
Now that you changed the file name maybe also change this title?

Reply:
Fixed: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/372/commits/7b0f0cb88712ee191cdcfd7b7c570dc0134af08e